packages/cli-tool/src/add-ons/ui-framework/tailwind.ts

```diff
@@ -0,0 +1,100 @@
+import * as fs from 'node:fs';
+
+import { cli } from 'cli-ux';
+
+import runCommand from '../../helpers/child-process';
+import { replaceLine } from '../../helpers/file-editor';
+
+const NPM_PACKAGES = [
+  'tailwindcss@^3.1.4',
+  'postcss@^8.4.14',
+  'postcss-import@^14.1.0',
+];
+
+const installNpmPackages = (appName: string): Promise<boolean> => {
+  return runCommand('npm', ['install', '-D', ...NPM_PACKAGES], `./${appName}/`);
```

Comment:
Curiosity, why are these packages flagged as `-D`

Reply:
Because Tailwind packages are dev utilities to parse HTML/JSX files and identify what CSS to include in the `@tailwind` helpers. So once built, you do not need these packages for runtime. In other words, you can think of these packages as "assets pipeline utilities" — i.e. purely dev dependencies.